DOCS/faq.md

```diff
@@ -36,127 +12,108 @@ standardized short identifier, the full name, the license text, and a
+## How does one handle licenses not found in the SPDX License List?
 
-Disjunctive licensing can be represented via a license expression using
-the OR operator. For example, a file that is disjunctively licensed
+When a license identified in the software package is not found in the
+list of approved SPDX licenses, one can define a new license label, using `LicenseRef-<name>`..This is explained in [Clause 10 of the SPDX Specification](https://spdx.github.io/spdx-spec/v2.3/other-licensing-information-detected/).
+
+# What are license expressions?
+Sometimes a single license can be used to represent the licensing terms of a package or file, but there are situations where a single license identifier is not sufficient. [SPDX License Expressions](https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/) provide a way to construct more complex licensing scenarios. expressions. A license expression could be a single license identifier found on the SPDX License List; a user defined license reference denoted by the LicenseRef-<idString>; a license identifier combined with an SPDX exception; or some combination of license identifiers, license references and exceptions constructed using a small set of defined operators (e.g., AND, OR, WITH and +)   
+ 
+## How does one represent a file or package that is licensed under a choice of two or more licenses?
+A licensing choice can be represented by a license expression using
+the `OR` operator. For example, a file that is disjunctively licensed
 under either the GPL-2.0-only or MIT would be represented using the following
 disjunctive expression: `GPL-2.0-only OR MIT`.
 
-##### <a href="#top"> Back to Top</a>
-
-<br>
-
-### <span id="multi"></span>How does one represent a file that is licensed under two or more licenses?
+## How does one represent a file or package that is licensed under two or more licenses?
 
 Conjunctive licensing can be represented via a license expression using
-the AND operator. For example, a file that is subject to the Apache-2.0,
+the 'AND' operator. For example, a file or package that is subject to the Apache-2.0,
 MIT, and GPL-2.0-or-later would be represented using the following conjunctive
-expression: `Apache-2.0 AND MIT AND GPL-2.0-or-later`.
-
-##### <a href="#top"> Back to Top</a>
-
-<br>
-
-## <span id="add"></span>Adding a License?
-
-### <span id="request"></span>How do I request adding a license to SPDX License List?
-
-Follow the instructions here:
-https://github.com/spdx/license-list-XML/blob/main/CONTRIBUTING.md
-
-##### <a href="#top"> Back to Top</a>
-
-<br>
+expression: `Apache-2.0 AND MIT AND GPL-2.0-or-later`
+ 
+# How are license identifiers used in an SPDX Document?
+The SPDX specification includes fields to record license information at the [package}(https://spdx.github.io/spdx-spec/v2.3/package-information/), [file](https://spdx.github.io/spdx-spec/v2.3/file-information/), and [snipped](https://spdx.github.io/spdx-spec/v2.3/snippet-information/) level. 
```

Comment:
Two things:
* there's a curly bracket after `package` rather than a square bracket -- this breaks the rendering
* typo for the final link, `snipped` => `snippet`

Reply:
fixed